src/redis_request.h

```diff
@@ -55,7 +55,7 @@ class Request {
   int64_t multi_bulk_len_ = 0;
   size_t bulk_len_ = 0;
   CommandTokens tokens_;
-  std::vector<CommandTokens> commands_;
+  std::list<CommandTokens> commands_;
```

Comment:
Do you mean deque will performance better than list in this case?

Reply:
Yeah, I think `std::deque` is more cache-friendly and its `pop/push_front/back` is also constant in complexity.

Reply:
After taking a look at how `std::deque`, I think you are right.
I thought `std::deque` will cause memory realloc since my memory
tells me it's an array type container. Will update this.